src/scikit_build_core/build/wheel.py

```diff
@@ -67,6 +67,7 @@ def _make_editable(
         verbose=settings.editable.verbose,
         build_options=build_options,
         install_options=install_options,
+        install_dir=settings.wheel.install_dir or None,
```

Comment:
```suggestion
        install_dir=settings.wheel.install_dir,
```
It should be fine to pass the default, it would even simplify the `install_dir or ""`  in my other comment